src/applications/personalization/profile/mocks/server.js

```diff
@@ -85,6 +85,42 @@ const logRequest = req => {
 };
 
 const responses = {
+  'GET /v0/in_progress_forms/mock-form-ae-design-patterns': (_req, res) => {
+    return res.json({
+      formData: {
+        data: {
+          attributes: {
+            veteran: {
+              address: {
+                addressLine1: '623 Lesser Dr',
+                city: 'Fort Collins',
+                stateCode: 'CO',
+                zipCode5: '80524',
+                countryName: 'USA',
+              },
+              firstName: 'John',
+              lastName: 'Donut',
+              middleName: 'Jelly',
+              phone: {
+                areaCode: '970',
+                phoneNumber: '5561289',
+              },
+              emailAddressText: 'sample@email.com',
+            },
+          },
+        },
+        nonPrefill: {
+          veteranSsnLastFour: '3607',
+          veteranVaFileNumberLastFour: '3607',
+        },
+      },
+      metadata: {
+        version: 0,
+        prefill: true,
+        returnUrl: '/veteran-details',
+      },
+    });
+  },
```

Comment:
same as above, we can remove this change in the profile code.

Reply:
done